test/azuremetadata/generated/Lro/README.md

```diff
@@ -37,6 +37,8 @@ const { AutoRestLongRunningOperationTestService } = require("");
 const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
 
 const creds = new DefaultAzureCredential();
+// Pass the credentials from `@azure/identity` to the client constructor.
+// Please note that the credentials from `@azure/ms-rest-nodeauth` are supported here as well.

```

Comment:
Just a thought...
Do you think it is better to have these comments above the line where the creds are created rather than above the client constructor? In that case, it would be

> Create credentials using the `@azure/identity` package.
Please note that you can also use credentials from the `@azure/ms-rest-nodeauth` package instead.

Reply:
@ramya-rao-a Yeah I like it